core/server/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -183,11 +185,13 @@ public static void beforeClass() throws Exception {
 
   @AfterClass
   public static void afterClass() throws Exception {
```

Comment:
Because here Configuration change is only done in `BeforeClass`. Configuration is not changed in each test case. We can also switch to `after`.